src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientConfiguration.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# New-AzureRmVpnClientConfiguration
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+```
+New-AzureRmVpnClientConfiguration -ResourceGroupName <String> -VirtualNetworkGatewayName <String>
+ [-ProcessorArchitecture <String>] -AuthenticationMethod <String> [-RadiusRootCert <String>]
+ [-ClientRootCertificates <String>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
+
+## PARAMETERS
+
+### -AuthenticationMethod
+Authentication Method
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+Accepted values: EAPTLS, EAPMSCHAPv2
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ClientRootCertificates
+Comma separated list of client root certificates
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ProcessorArchitecture
+ProcessorArchitecture
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+Accepted values: Amd64, X86
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -RadiusRootCert
+Radius server root certificate path
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+ResourceGroup name
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -VirtualNetworkGatewayName
+VirtualNetworkGateway name
```

Comment:
These parameter descriptions need to be complete.  This should be a complete sentence and describe